### ISSUE TITLE
Go shopping: Display which item you're shopping for

### ISSUE THREAD

**Issue by mrsimpson** [mrsimpson/collaborative-shopping#40]
## Motivation

In order to be motivated and maybe sort the items in my cart already, I want to be able to easily see for whom I'm shopping for

## Implementation proposal

- [x] Make the item list expandable / collapsible
- [x] When collapsed, display the avatars of all participants inline (overlapping to a large amount) right next to the name of the item
- [x] when expanded, display all separate quantities one in each line and display the avatar of the one you're shopping for.

**Comment by vbarzana** [comment]
This is another view proposal.
```
|----------------------------------|
| V Tomatoes                    20pc  |
|   - Mine                     4pc    |
|   - Oliver's                 10pc  |
|   - Steffi's                   6pc   |
| V Bananas                 20pc  |
|   - Mine                     4pc    |
|   - Oliver's                 10pc  |
|   - Steffi's                    6pc   |
|---------------------------------|
```

**Comment by mrsimpson** [comment]
Except the avatars, this has been implemented with #61
